addons/mail/static/src/components/composer/composer.js

```diff
@@ -207,7 +207,7 @@ class Composer extends Component {
             }
             return;
         }
-        await this.composer.postMessage();
+        await this.composer.insertMessageToBeSent();
```

Comment:
I have nothing remotely interesting to say here that would help or change anything except that we should not hide important technical detail of how it work. Using `postMessage` where it's actually `insertMessageInAQueue` is basically a lie.

Reply:
> We should not hide important technical detail of how it work. Using `postMessage` where it's actually `insertMessageInAQueue` is basically a lie.

I agree `postMessage` is a lie, and the problem comes from component having too much business code and the naming of actions dispatched to models is not correct. The user clicks on "Send" button, the models should receive "user click on button", not "post message".

I'm aware of this problem, and I plan to work on this ASAP. As I said in big comment above, should be done in or after my big PR... I'll make a big pass on all our components.

Maybe here we could have a semblance of what we want eventually. As I said in other comments, I don't like the name of public method in model. The misuses introduce many problems, such as: badly documented method, very long method (hundreds of LOCs), recursive method called both by component and internally in the model (in the method itself and somewhere else), etc. This will make proper refactoring hard as it is.

I suggest naming public method `onClickSend` of model. It's okay for the inner code of this function to actually "insert message to send in queue" if we feel it's the right technical way to act from user interaction.